### ISSUE TITLE
resource labels CRUD needs consistent interface

### ISSUE THREAD

**Issue by OfTheDelmer** [influxdata/influxdb2-js#22]
reviewing the methods from https://github.com/influxdata/influxdb2-js/issues/21 it looks like each labelable resource should have the following:

- `addLabel` with `(id: string, label: Label) => Promise<Label>`
-  `addLabels` with `(id: string, label: Label[]) => Promise<Label[]>`
-  `removeLabel` with `(id: string, label: Label) => Promise<Response>`
-  if can be cloned? `cloneLabels`  with `<Resource>(old: Resource, new: Resource) => Promise<Label[]>`  

Basically it just looks like this needs to only be applied to 

-  [ ] `dashboards`

**Comment by OfTheDelmer** [comment]
@bthesorceror after https://github.com/influxdata/influxdb2-js/pull/26 the naming now updated and consistent. Did we want to change the `LabelID` to `Label` as the second argument for dashboards?
